lib/PuppeteerSharp.Tests/PageTests/GotoTests.cs

```diff
@@ -22,8 +22,17 @@ public GotoTests(ITestOutputHelper output) : base(output)
             Assert.Null(response);
         }
 
+        [Fact]
+        public async Task ShouldNavigateToEmptyPageWithDOMContentLoaded()
+        {
+            var response = await Page.GoToAsync(TestConstants.EmptyPage, waitUntil: new WaitUntilNavigation[] {
```

Comment:
use `new[]` instead of `new WaitUntilNavigation[]`

Reply:
Done